modules/deploy/83-flowmanager-deploy.erb

```diff
@@ -5,7 +5,7 @@ kind: Deployment
 metadata:
   name: flowmanager
 spec:
-  replicas: 6
+  replicas: 2
```

Comment:
Revert this. (Or add an `if ENV["TF_VAR_cluster_name"] ...` stanza if you prefer; otherwise I'll fix all these up this week.)